style.css

```diff
@@ -1,340 +1,1403 @@
+    border-color: var(--color-blue);
+}
+
+.form-checkbox:has(.form-checkbox__input:checked):active .form-checkbox__box {
+    background-color: var(--color-blue);
+    border-color: var(--color-blue);
+}
+
+.form-checkbox:has(.form-checkbox__input:checked):active .form-checkbox__box::before {
+    filter: brightness(0) saturate(100%) invert(24%) sepia(93%) saturate(3065%) hue-rotate(204deg) brightness(92%) contrast(98%);
+}
+
+.form-checkbox:has(.form-checkbox__input:checked):focus .form-checkbox__box {
+    background-color: var(--color-blue);
+    border-color: var(--color-blue);
+}
+
+.form-checkbox__inner {
+    display: flex;
+    align-items: center;
+    justify-content: start;
+}
+
+.form-checkbox__inner a {
+    color: #78BBFA;
+}
+
+.form-checkbox__inner a:hover {
+    text-decoration: underline;
+}
+
+.form-checkbox__input:checked + .form-checkbox__inner .form-checkbox__box {
+    margin: 0 5px 0 0;
+    background-color: var(--color-blue);
+    border-color: var(--color-blue);
+}
+
+.form-checkbox__input:checked + .form-checkbox__inner .form-checkbox__box::before {
+    opacity: 1;
+}
+
+.form-checkbox__input:disabled + .form-checkbox__inner .form-checkbox__box {
+    background-color: #2e2f3c;
+    border-color: #2e2f3c;
+}
+
+.form-checkbox__input:invalid + .form-checkbox__inner .form-checkbox__box {
+    margin: 0 5px 0 0;
+    border-color: #D4D9F242;
+}
+
+.form-checkbox__box {
+    margin-top: 2px;
+    display: inline-block;
+    width: 14px;
+    height: 14px;
+    border-radius: 4px;
+    background-color: transparent;
+    border: 2px solid rgba(212, 217, 242, 0.26);
+    transition-property: border-color, background-color;
+    transition-duration: .2s;
+    position: relative;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.form-checkbox__box::before {
+    content: '';
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABkAAAATCAYAAABlcqYFAAAACXBIWXMAACE4AAAhOAFFljFgAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAADoSURBVHgBtdPtDYIwEAbgO4Oi4Q+O5CTqJIygTiAbMIpugol811h7DZLKl1DKmxQIbe6h6QFgMGnOvChjIY04f11Dzl0wGSoaZ4yrI8lYQHMIBkIAcH5omytsazsZ6QMoznqJC5gRAERf3mAmABHum5W1Q8SH3InaFeKwTmAQkC+oaL0rZJEeoLFe7aic3XitdVFMhOLutnyO79jWcdIOynQfvCim7kgXoFhvDpcFgtcDVc+gAch5ukRp4YsFe9DIP6BCdKEhwA8yFhoKNJCh0BigMwSN+Q+MQkaBb54JO5dAKEZgHDCZD1zmI/Su9qWCAAAAAElFTkSuQmCC');
+    background-position: center;
+    background-size: 8px 6px;
+    background-repeat: no-repeat;
+    opacity: 0;
+    transition: opacity .2s ease;
+}
+
+.form-checkbox__text {
+    color: #F5F8FB;
+    font-size: 14px;
+    line-height: 1.4em;
+}
+
+/* form switcher */
+
+.form-switcher {
+    padding: 24px;
+    display: flex;
+    justify-content: space-between;
+    background-color: #1D1C28;
+}
+
+.form-switcher__text {
+    font-size: 14px;
+    line-height: 1.4em;
+    color: #F5F8FB;
+}
```

Comment:
Delete this block. All this properties should be on the .contact__form

Reply:
done